### PR TITLE
Forward port release-1.0 into master

### PR DESCRIPTION
**Pull request by davidebriani**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,13 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.0.0-beta.2] - Unreleased
```
Comment (Pavinati) [inline CHANGELOG.md:7]: this is wrong, [Unreleased] contains the changelog of master (where we reworked the interface editor)
`## [1.0.0-beta.2] - Unreleased` is where changed the login layout

**Review by Pavinati — CHANGES_REQUESTED** [review]

File: cypress/integration/blocks_page_test.js [hunk 2]
```diff
@@ -64,4 +63,4 @@ describe('Blocks page tests', () => {
       cy.location('pathname').should('eq', `/blocks/${this.blocks.data[0].name}/edit`);
     });
   });
-});
+});
```
Comment (Pavinati) [inline cypress/integration/blocks_page_test.js:66]: missing newline

File: cypress/integration/device_page_test.js [hunk 3]
```diff
@@ -837,4 +884,4 @@ describe('Device page tests', () => {
       });
     });
   });
-});
+});
```
Comment (Pavinati) [inline cypress/integration/device_page_test.js:887]: missing newline

**Review by Pavinati — CHANGES_REQUESTED** [review]
maybe we should add a linter rule for that (eol-last)

**Review by Pavinati — APPROVED** [review]

**Review by bettio — APPROVED** [review]

**Pull request merged**